assets/js/modules/analytics/components/settings/SettingsView.js

```diff
@@ -16,21 +16,36 @@
  * limitations under the License.
  */
 
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
 /**
  * Internal dependencies
  */
 import Data from 'googlesitekit-data';
 import { ExistingGTMPropertyNotice } from '../common';
+import DisplaySetting from '../../../../components/DisplaySetting';
 import StoreErrorNotices from '../../../../components/StoreErrorNotices';
 import GA4SettingsView from './GA4SettingsView';
 import UASettingsView from './UASettingsView';
 import OptionalSettingsView from './OptionalSettingsView';
-import { MODULES_ANALYTICS } from '../../datastore/constants';
+import {
+	DASHBOARD_VIEW_GA4,
+	MODULES_ANALYTICS,
+} from '../../datastore/constants';
 import { MODULES_TAGMANAGER } from '../../../tagmanager/datastore/constants';
 import { CORE_MODULES } from '../../../../googlesitekit/modules/datastore/constants';
+import { useFeature } from '../../../../hooks/useFeature';
 const { useSelect } = Data;
 
 export default function SettingsView() {
+	const ga4ReportingEnabled = useFeature( 'ga4Reporting' );
+	const isGA4Connected = useSelect( ( select ) =>
+		select( CORE_MODULES ).isModuleConnected( 'analytics-4' )
+	);
+
```

Comment:
Same here. Please refer to the above comment about using the `isGA4DashboardView` selector.

Reply:
Again, `ga4ReportingEnabled()` is checked separately to `isGA4Connected()` in this component, so `isGA4DashboardView()` would not be a useful replacement (plus the question about changing the logic in `isGA4DashboardView()` also stands).